framework/test-vectors/esdk-test-vector-enumeration.md

```diff
@@ -0,0 +1,59 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# ESDK Test Vector Enumeration
+
+This document performs the [test vector enumeration](test-vector-enumeration.md) process for the ESDK
+to construct the full suite of test vectors
+for the ESDK.
+
+The full suite of test vectors can be constructed
+by [enumerating all input configurations](test-vector-enumeration.md#enumerating-input-configurations) from this spec's input dimensions
+and [evaluating each configuration's expected result](test-vector-enumeration.md#determining-expected-results) from this spec's evaluation rules.
+
+## Input dimensions
+
+- Every [MPL input dimension](mpl-test-vector-enumeration.md#input-dimensions) is an input dimension for ESDK.
+- plaintext: Range of [representative plaintext values](#representative-plaintext-constraints)
+- commitment policy: Range of allowed [commitment policies](../../client-apis/client.md#commitment-policy)
+- frame size: Range of [representative frame sizes](#representative-frame-sizes)
+
+## Evaluation rules
+
+- Every [MPL evaluation rule](mpl-test-vector-enumeration.md#evaluation-rules) is an evaluation rule for ESDK.
+
+## Representative values
+
+### Representative frame sizes
+
+- Non-framed data: `length = 0`
+- Small frame: `0 < length < 4096`
+- Default frame: `length = 4096`
+- Large frame: `length > 4096`
+
+### Representative plaintext constraints
+
+#### Framed data
```

Comment:
it may be easy to add MUSTs to these and clearly duvet them in the implementation, this way it is easy to thread these "constants" and trace them back to this part of the spec